### PR TITLE
[WIP]One_to_many_relationship

### PR DESCRIPTION
**Pull request by nawal-neeraj**
image (4)

### CODE REVIEW

File: blogs/Database/database.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import os
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import declarative_base
+
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+userName: str = "nawal"
```
Comment (rakeshcusat) [inline blogs/Database/database.py:12]: python variable naming convention?

File: blogs/hasher/hash.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from passlib.context import CryptContext
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+def hashPasswor(password):
```
Comment (rakeshcusat) [inline blogs/hasher/hash.py]:
```suggestion
def hash_password(password):
```

File: blogs/schema/model.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+from typing import List, Optional
+from pydantic import BaseModel
+
+class Blogs(BaseModel):
+    id: Optional[int]= None
+    title: str
+    body: str
+
+class Show_user(BaseModel):
```
Comment (rakeshcusat) [inline blogs/schema/model.py]:
```suggestion
class ShowUser(BaseModel):
```
follow a consistent naming convention.

File: blogs/schema/model.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    class Config():
+        orm_mode = True
+
+class Users(BaseModel):
+    id: Optional[int]= None
+    name: str
+    email: str
+    password: str
+    
+class user_resp(BaseModel):
```
Comment (rakeshcusat) [inline blogs/schema/model.py]:
```suggestion
class UserResp(BaseModel):
```

File: blogs/schema/model.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    id: int
+    name: str
+    email: str
+    class Config():
+        orm_mode = True
+    
+    
+class Show_blogs(BaseModel):
```
Comment (rakeshcusat) [inline blogs/schema/model.py]:
```suggestion
class ShowBlogs(BaseModel):
```

File: blogs/Database/database.py (hunk 1 above)
Comment (nawal-neeraj) [reply blogs/Database/database.py:12]: made changes according to comment

**Pull request merged**